doc/modules/compose.rst

```diff
@@ -5,14 +5,23 @@
 Pipelines and composite estimators
 ==================================
 
-Transformers are usually combined with classifiers, regressors or other
-estimators to build a composite estimator.  The most common tool is a
-:ref:`Pipeline <pipeline>`. Pipeline is often used in combination with
-:ref:`FeatureUnion <feature_union>` which concatenates the output of
-transformers into a composite feature space.  :ref:`TransformedTargetRegressor
+To build a composite estimator, transformers are usually combined with other
+transformers or with :term:`predictors` (such as classifiers or regressors).
+The most common tool used for this is a :ref:`Pipeline <pipeline>`. Pipelines
+require all steps except the last to be a :ref:`transformer`. The last step can
```

Comment:
I suspect that this is causing the Sphinx warning that makes the circle CI doc fail with the following message:

```
doc/modules/compose.rst:8: WARNING: undefined label: 'transformer' 
```
https://output.circle-artifacts.com/output/job/2189993c-3da9-4db8-809b-bf4564864bcd/artifacts/0/doc/_changed.html

```suggestion
require all steps except the last to be a :term:`transformer`. The last step can
```

Reply:
I see, thank you.